2766. Relocate Marbles/solution.py

```diff
@@ -0,0 +1,21 @@
+class Solution:
+    # Given a list of marbles (nums), this function returns a list of the occupied positions of the
+    # marbles. Each element in nums represents a position, and you are supposed to move each marble
+    # in that position 
+    def relocateMarbles(self, nums: List[int], moveFrom: List[int], moveTo: List[int]) -> List[int]:
+        # iterate through moveFrom and moveTo.
+            
+            # for each number in nums...
+                
+                # if it's moveFrom, move it to moveTo.
```

Comment:
It's ambiguous what operation you will do here

Reply:
What do you mean?